cmake/Intel.cmake

```diff
@@ -16,16 +16,9 @@ if(DEBUG)
       endif()
     endif()
     set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -O0 -ftrapuv")
-elseif(REPRO)
-    add_definitions(-DREPRO)
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -O2 -debug minimal -fp-model consistent -qoverride-limits")
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O2 -debug minimal")
 else()
-    if(32BIT)
-      set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -O2 -debug minimal -fp-model source -qoverride-limits -qopt-prefetch=3")
```

Comment:
I think you may also want to remove
```
CMAKE_C_FLAGS_BITFORBIT
CMAKE_Fortran_FLAGS_BITFORBIT
CMAKE_CXX_FLAGS_BITFORBIT
```
in `Intel.cmake` as well as `GNU.cmake`

Reply:
It would also be nice if `CMAKE_*_FLAGS_RELEASE` would be set and used by the ufs-weather-model itself, not just for CPPP.

Reply:
Done. See https://github.com/ufs-community/ufs-weather-model/pull/1171/commits/44ef119ef2bbde67f331420ba85bf87aaac34992

Reply:
@climbfuji Do you remember why ccpp needs -fPIC? Is it still needed?

Reply:
Don't think it is needed the way CCPP is used nowadays (compiled as part of the executable). In the early days, the idea was to compile standalone libraries that could be used by whatever application.

Reply:
Thanks. Removed in https://github.com/ufs-community/ufs-weather-model/pull/1171/commits/3ab2732f05235500b58a2287a985e7ad6ae1c62d